index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Portfolio-Mobile-First</title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width,initial-scale=1">
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <div class="wrapper">
+            <header>
+                <div class="logo">Welcome</div>
```

Comment:
- [OPTIONAL] I would recommend you to make this logo a link to the home page. That is the standard way in any website you see online. It is marked optional because it was not mentioned by the previous reviewer, but it is highly recommended you attend to it.